pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -32,15 +33,15 @@ type CreateBootstrapPolicyFileOptions struct {
 	OpenShiftSharedResourcesNamespace string
 }
 
-func NewCommandCreateBootstrapPolicyFile() *cobra.Command {
+func NewCommandCreateBootstrapPolicyFile(commandName string, baseName string, out io.Writer) *cobra.Command {
```

Comment:
Seems like we'd want to pass name and baseName as opposed to name and fullName.  Why would we want fullName instead?